gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,3 +1,4 @@
+#Sat Apr 04 23:50:58 BOT 2020
```

Comment:
Undo changes in this file.

Reply:
If I do I can't compile, I get the error Minimum supported Gradle version is 5.6.4